tests/default/data/unresolvable-pdo-statement.php

```diff
@@ -38,5 +42,9 @@ public function noErrorOnStringValue(PDO $pdo, string $string)
         $query = 'SELECT adaid FROM ada WHERE email=:email';
         $stmt = $pdo->prepare($query);
         $stmt->execute([':email' => '%|'.$string.'|%']);
+
+        $stmt = $pdo->prepare($query);
+        $stmt->bindValue(':email', '%|'.$string.'|%');
+        $stmt->execute();
```

Comment:
I think we can/should cover a case where some params are bound via `bindValue` or `bindParam` and others are given to `execute` at the same time.. I guess this is supported in pdo?

Reply:
would be cool if you could run a small example on your own and verify the thesis.
if its right, we might even create a PHPStan-Rule which errors on `bindValue`/.. calls, when parameters are passed to `execute`

Reply:
Cool, thx

https://github.com/staabm/phpstan-dba/issues/275